src/main/java/com/linkedin/kmf/apps/MultiClusterMonitor.java

```diff
@@ -34,38 +35,34 @@
   private final ConsumeService _consumeService;
   private final String _name;
 
-  public MultiClusterMonitor(Map<String, Object> props, String name) throws Exception {
+  public MultiClusterMonitor(Config serviceConfig, String name) throws Exception {
     _name = name;
-    MultiClusterMonitorConfig config = new MultiClusterMonitorConfig(props);
-    _topicManagementService = new MultiClusterTopicManagementService(createMultiClusterTopicManagementServiceProps(props, config), name);
-    _produceService = new ProduceService(createProduceServiceProps(props, config), name);
-    _consumeService = new ConsumeService(createConsumeServiceProps(props, config), name);
+    MultiClusterMonitorConfig config = new MultiClusterMonitorConfig(serviceConfig);
+    _topicManagementService = new MultiClusterTopicManagementService(createMultiClusterTopicManagementServiceProps(serviceConfig, config), name);
```

Comment:
since there are new commits to the _master_ branch, let's **resolve** the **conflicts** for better review.